css-ui-4/Overview.bs

```diff
@@ -1551,6 +1556,10 @@ an <dfn>editable element</dfn> is either
 an <a href="https://w3c.github.io/editing/contentEditable.html#dfn-editing-host">editing host</a>
 or a <a href="https://www.w3.org/TR/html/sec-forms.html#mutable">mutable</a> form control with textual content,
 such as <{textarea}>.
+A <dfn>locked editable element</dfn> is
+a normally <a href="https://www.w3.org/TR/html/sec-forms.html#mutable">mutable</a> form control with textual content,
+when it is placed in a non mutable state,
+such as a <{textarea}> with the <a href="https://html.spec.whatwg.org/multipage/input.html#the-readonly-attribute"><code>readonly</code></a> attribute.
```

Comment:
It seems to me that this definition is probably not clear enough, for example, it is unclear whether a disabled form control is a "locked editable element". I guess the ambiguity is from "normally mutable form control". Is a disabled form control considered normally mutable? Or is being disabled just a non-mutable state?

Maybe explicitly referencing `readonly` attribute rather than listing it as an example may help? Something like "a otherwise mutable form control ... when it is placed in a non-mutable state by readonly attribute".